RealmSwift-swift3.0-dev/List.swift

```diff
@@ -31,7 +31,7 @@ public class ListBase: RLMListBase {
         return descriptionWithMaxDepth(depth: RLMDescriptionMaxDepth)
     }
 
-    @objc private func descriptionWithMaxDepth(depth: UInt) -> String {
+    @objc(descriptionWithMaxDepth:) private func descriptionWithMaxDepth(depth: UInt) -> String {
```

Comment:
do Swift `@objc` methods get de-clang-imported? In other words, would `description(maxDepth: UInt)` infer the right selector?